handlers.go

```diff
@@ -141,6 +147,30 @@ func (i *Impl) GET_Faults(rw http.ResponseWriter, req *http.Request, _ httproute
 	rw.Write(js)
 }
 
+func (i *Impl) getLatestFaults() map[string]*FaultEntry {
```

Comment:
This isn't ideal and very computationaly expensive but for demo purposes of this prototype this should be good enough